Libraries/Components/Slider/Slider.js

```diff
@@ -115,6 +116,24 @@ var Slider = React.createClass({
     thumbImage: Image.propTypes.source,
 
     /**
+     * Color of the foreground switch grip.
+     * @platform android
+     */
+    thumbTintColor: ColorPropType,
+
+    /**
+     * Color of the slider track.
+     * @platform android
+     */
+    trackTintColor: ColorPropType,
```

Comment:
I'm not sure, I never used those props but I think that's what it does. If it is the case we could use your name and deprecate those, I agree that it is not really clear. It would just be nice to be able to set that using the same prop for ios and android.